portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/uiinfrastructure/frontenddataset/datamanagment/DatasetsFilters.testcase

```diff
@@ -0,0 +1,214 @@
+@component-name = "portal-frontend-infrastructure"
+definition {
+
+	property custom.properties = "feature.flag.LPS-164563=true";
+	property portal.acceptance = "true";
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Frontend Dataset";
+	property testray.main.component.name = "Frontend Dataset";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		task ("Given creating a Data Set via API") {
+			FrontendDataSetAdmin.goToDatasetAdminPage();
+
+			FrontendDataSetAdmin.createDatasetEntryViaAPI(
+				datasetName = "Dataset Test",
+				restApplication = "/c/fdsdatefilters",
+				restEndpoint = "/",
+				restSchema = "FDSDateFilter");
+		}
+
+		task ("And a dataset view created via api") {
+			FrontendDataSetAdmin.createFDSViewViaAPI(
+				datasetName = "Dataset Test",
+				dataSetViewDescription = "Description Test",
+				key_dataSetViewNameList = "View Test");
+		}
+
+		task ("When go to the View page of the dataset created.") {
+			FrontendDataSetAdmin.goToSpecificViewPage(
+				datasetName = "Dataset Test",
+				key_viewName = "View Test");
+		}
+
+		task ("When the user accesses the Filters tab.") {
+			FrontendDataSetAdmin.goToTab(tabName = "Filters");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		FrontendDataSetAdmin.deleteAllDatasetEntries();
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@description = "LPS-183476. Confirm that all fields in the schema are displayed"
+	@priority = 5
+	test AssertAllFieldsAreDisplayedInSchema {
+		task ("And clicks to add a new filter") {
+			LexiconEntry.gotoAdd();
+		}
+
+		task ("asserts all fields in the schema are displayed") {
+			Click(locator1 = "FrontendDataSetAdmin#FILTER_BY_SELECT");
+
+			FrontendDataSetAdmin.assertSearchFilters(key_filterList = "creator,externalReferenceCode,id,status,createDate,fieldName,from,modifiedDate,name,r_fdsViewFDSDateFilterRelationship_c_fdsViewId,rendererType,r_fdsViewFDSDateFilterRelationship_c_fdsViewERC,to,type");
+		}
+	}
+
+	@description = "LPS-183476. Confirm that when 'Name Field' is not provided, it is filled with the same name as field name."
```

Comment:
Put the correct LPS--> LPS-176184 (in Poshi, we should put the LPS of the original story/bug)